src/screens/Settings/CryptoAssets/Rates/RatesList.js

```diff
@@ -0,0 +1,157 @@
+/* @flow */
+import React, { PureComponent } from "react";
+import { StyleSheet, ScrollView, View } from "react-native";
+// $FlowFixMe
+import { FlatList } from "react-navigation";
+import { connect } from "react-redux";
+import i18next from "i18next";
+import { Trans } from "react-i18next";
+import { createStructuredSelector } from "reselect";
+import type { NavigationScreenProp } from "react-navigation";
+import type { Currency } from "@ledgerhq/live-common/lib/types";
+import { pairsSelector } from "../../../../countervalues";
+import SettingsRow from "../../../../components/SettingsRow";
+import LText from "../../../../components/LText";
+import colors from "../../../../colors";
+import Price from "./Price";
+
+type Pair = {|
+  from: Currency,
+  to: Currency,
+  exchange: ?string,
+|};
+
+type Props = {|
+  navigation: NavigationScreenProp<*>,
+  pairs: Pair[],
+|};
+
+const mapStateToProps = createStructuredSelector({
+  pairs: pairsSelector,
+});
+
+const currencyColors = {
+  fiat: "#6490f1",
+  crypto: "#41ccb4",
+};
+
+class CurrenciesList extends PureComponent<Props> {
+  static navigationOptions = () => ({
+    title: i18next.t("settings.rates.header"),
+  });
+
+  renderItem = ({ index, item: p }: { index: number, item: Pair }) => (
+    <SettingsRow
+      title={
+        <Trans i18nKey="settings.rates.pair">
+          {{ from: p.from.ticker }}
+          {{ to: p.to.ticker }}
+        </Trans>
+      }
+      titleContainerStyle={[
+        styles.rowTitle,
+        {
+          borderLeftColor:
+            p.to.type === "FiatCurrency"
+              ? currencyColors.fiat
+              : currencyColors.crypto,
+        },
+      ]}
+      desc={<Price from={p.from} to={p.to} style={styles.rowDesc} />}
+      noTextDesc
+      borderTop={index !== 0}
+      arrowRight
+      compact
+      event="RateSettingsRow"
+      onPress={() =>
+        this.props.navigation.navigate("RateProviderSettings", {
+          from: p.from.ticker,
+          to: p.to.ticker,
+          selected: p.exchange,
+        })
+      }
+    >
+      {/* FIXME display actual name instead of ID */}
+      <LText>{p.exchange || ""}</LText>
+    </SettingsRow>
+  );
+
+  // prettier-ignore
+  keyExtractor = ({ from, to }: Pair): string => `${from.name || ""}-${to.name || ""}`;
+
+  render() {
+    const { pairs } = this.props;
+
+    return (
+      <ScrollView style={styles.root}>
```

Comment:
You should use FlatList's `ListHeaderComponent` prop rather than nesting scrollables